src/tools/rest-frontend/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "rest-frontend",
+  "version": "1.0.0",
+  "description": "rest frontend application",
+  "main": "Gruntfile.js",
+  "keywords": [
+    "elektra",
+    "kdb",
+    "frontend",
+    "rest",
+    "snippet sharing"
+  ],
+  "author": "Namoshek",
+  "license": "BSD",
+  "devDependencies": {
```

Comment:
This is a very good question actually. I'm myself not entirely sure what the better option to use is, but because dependencies are only required to "build" the frontend (some deps are copied then), they could be considered dev-dependencies.

I think it doesn't really matter in this case, as the installation works exactly the same for both types. Dev-dependencies are just left out in case one invokes `npm install` with `--production`, what is not explained/recommended anywhere in my docs.